database/index.js

```diff
@@ -0,0 +1,56 @@
+const Sequelize = require('sequelize');
+const seed = require('./seed.js');
+
+console.log(seed.seed)
+
+// Option 1: Passing parameters separately
+const sequelize = new Sequelize('yelpPhotoGallery', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+  define: {
+    // The `timestamps` field specify whether or not the `createdAt` and `updatedAt` fields will be created.
+    // This was true by default, but now is false by default
+    timestamps: false
+  }
+});
+
+sequelize
```

Comment:
sequelize.authenticate() and users.sync() are async functions. you may want to have your models sync only after you have authenticated your connection by linking your promises chains